src/_components/home/hero/card.erb

```diff
@@ -0,0 +1,7 @@
+<home-hero-card>
+  <a href="<%= link %>">
+    <img src="<%= relative_url "images/#{@card.icon}.svg" %>">
```

Comment:
I don't have anything generic to put in there, however  for now I'm pretty sure this will be seen during review if we were in a case where an icon is missing.